packages/scene-composer/src/components/panels/scene-components/common/ValueDataBindingBuilder.tsx

```diff
@@ -193,14 +204,53 @@ export const ValueDataBindingBuilder: React.FC<IValueDataBindingBuilderProps> =
               }
               key={definition.fieldName}
             >
-              <Select
-                data-testid='value-data-binding-builder-select'
-                selectedOption={selectedOption}
-                onChange={onSelectChange(definition.fieldName, index)}
-                options={options}
-                disabled={disabled}
-                placeholder={intl.formatMessage({ defaultMessage: 'Select an option', description: 'placeholder' })}
-              />
+              {definition.fieldName === 'propertyName' && (
+                <div className='copy-select-flex'>
```

Comment:
Please refer to this https://cloudscape.aws.dev/components/space-between/?tabId=playground. You can use this to place them in the same line. And also you don't need to have two separate select as you can toggle copy button based on the definition.